src/components/organisms/AssetActions/Pool/index.tsx

```diff
@@ -170,11 +172,15 @@ export default function Pool({ ddo }: { ddo: DDO }): ReactElement {
       }
     }
     init()
-  }, [ocean, accountId, price, ddo.dataToken, refreshPool, ddo.publicKey])
+
+    // Re-fetch price periodically, triggering re-calculation of everything
+    const interval = setInterval(() => refreshPrice(), refreshInterval)
+    return () => clearInterval(interval)
+  }, [ocean, accountId, price, ddo, refreshPool])
 
   const refreshInfo = async () => {
     setRefreshPool(!refreshPool)
-    await refreshPrice()
```

Comment:
why remove `await`?

Reply:
editor told me it's useless cause `refreshPrice` is no async method. But could add it back if there was a reason for it?

Reply:
this is interesting ` async function refreshPrice(): Promise<void>` , could it be because it returns `Promise<void>`, but i use `async` functions in it so it needs to be awaited

Reply:
not sure, I put it back for now. editor shows me this:
<img width="390" alt="Screen Shot 2020-11-02 at 11 48 30" src="https://user-images.githubusercontent.com/90316/97859782-5e591000-1d01-11eb-994d-d3b2c6f8bce5.png">